WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -450,6 +450,8 @@ import Foundation
 
     // Help & Support
     case supportOpenMobileForumTapped
+    case supportMigrationFAQTapped
+    case supportMigrationFAQViewed
```

Comment:
nit: I first thought "viewed" meant the user had viewed the FAQ. It could be a bit more verbose like:
```
    case supportMigrationFAQCardViewed
    case supportMigrationFAQButtonTapped